typescript/infra/config/environments/mainnet3/warp/configGetters/getBlastZeroNetworkUSDBWarpConfig.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ChainMap,
+  IsmType,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import { safes } from '../../owners.js';
+
+const USDB_ADDRESS = '0x4300000000000000000000000000000000000003';
+
+export const getBlastZeroNetworkUSDBWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const blast: TokenRouterConfig = {
+    ...routerConfig.blast,
+    type: TokenType.collateral,
+    token: USDB_ADDRESS,
+    interchainSecurityModule: {
```

Comment:
is this not able to be set to `ethers.constants.AddressZero`?
imo we should much prefer this over the fallback routing hook

Reply:
Will leave this as is until we fix https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/4871

Reply:
could we actually just do this now?

I think it's fine for this to be inconsistent with what the CLI does - I doubt we'll wanna update this later because it requires multisig interactions so imo there's no downside to moving over to the zero address now